### PR TITLE
Update terraform version

### PR DESCRIPTION
**Pull request by FlavioF**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM golang:alpine
 
-ENV TERRAFORM_VERSION 0.11.8
+ENV TERRAFORM_VERSION 0.11.10
 ENV CLOUD_SDK_VERSION 217.0.0
```
Comment (jeff-knurek) [inline Dockerfile]: maybe update this version too while you're at it...? if not make another PR after

Comment (FlavioF) [reply Dockerfile]: I will add another PR for that ok? :)
